cwolfmap/audio.c

```diff
@@ -159,7 +159,11 @@ int CWAudioLoadAudioT(CWAudio *audio, const CWMapType type, const char *path)
 
 	free(audio->data);
 
-	const uint32_t len = audio->head.offsets[audio->head.nOffsets - 1];
+	uint32_t len = audio->head.offsets[audio->head.nOffsets - 1];
+#if (SDL_BYTEORDER == SDL_BIG_ENDIAN)
```

Comment:
I'd like to keep cwolfmap independent of SDL (test programs can use SDL though), so can you replace these with an alternative? I think https://github.com/malcom/ByteOrder looks good, it has a permissive license and is easy to use (we would use `letoh...` functions)

Reply:
ok i test that, i put byteorder twice (dir WAD seem independant ?!).. let me know if not good